### PR TITLE
Fabo/handle batch failure

### PR DESCRIPTION
**Pull request by faboweb**
Batch transactions would fail partially but they would show as a success

### CODE REVIEW

File: api/lib/reducers/polkadotV0-reducers.js [hunk 1]
```diff
@@ -243,15 +243,26 @@ function parsePolkadotTransaction(
...
+  .filter(({ phase }) => {
+    parseInt(phase.toHuman().ApplyExtrinsic) === extrinsicIndex // index is a string
+  })
+  // if tx is a batch, we need to check if all of the batched txs went through
+  if (isBatch) {
+    return !!events
+      .find(
+        ({ event }) =>
+          event.section === `utility` &&
+          event.method === `BatchCompleted`
```
Comment (faboweb) [inline api/lib/reducers/polkadotV0-reducers.js:257]: We should invert here and directly look for the errors so we can get the error codes

**Comment by jbibla** [comment]
can you please run `lint --fix`

File: api/lib/reducers/polkadotV0-reducers.js (hunk 1 above)
Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:257]: mmm, something seems to be failing here

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js:257]: image

Comment (Bitcoinera) [reply api/lib/reducers/polkadotV0-reducers.js]: Fixed

**Comment by faboweb** [comment]
Great, thanks @Bitcoinera

**Pull request merged**